server/src/com/scraniel/auctioneero/Bootstrapper.java

```diff
@@ -0,0 +1,45 @@
+package com.scraniel.auctioneero;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+
+import org.hibernate.*;
+
+import com.scraniel.auctioneero.hbm.HibernateContext;
+import com.scraniel.auctioneero.tables.User;
+
+public class Bootstrapper 
+{
+
+	public static void main(String[] args) 
+	{	
+		HibernateContext context = HibernateContext.getInstance();
+		try
+		{
+			// Bootstrap hibernate so we can run some queries			
```

Comment:
You can setup test framework to validate different scenarios. Could be a good learning here as well :)

Reply:
Yep. I believe maven has test integration as well so will be nice to take a look at